cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,19 @@
+from django.core.management.base import BaseCommand
+from django.db.utils import OperationalError
+from django.db import connections
+
+import time
```

Comment:
Order your imports [accroding to PEP-8](https://peps.python.org/pep-0008/#imports). `time` is a standard library